src/Details.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+import './App.css';
+
+class Details extends Component {
```

Comment:
Excellent component but might want to give this a better name like OrganisationDetail